src/Appium.Net/Appium/Interfaces/IHasPerformanceData.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+
+namespace OpenQA.Selenium.Appium.Interfaces
+{
+    public interface IHasSupportedPerformanceData
+    {
+        /// <summary>
+        /// Returns the information of the system state which is supported to read as like cpu, memory, networ
+        /// </summary>
+        /// <param name="packageName">The package name of the application</param>
+        /// <param name="dataType">The type of system state which wants to read. It should be one of the supported performance data types.</param>
```

Comment:
Please enumerate the supported data types explicitly. Perhaps, it would make sense to have a separate enum for these